### PR TITLE
feat(guides-track): track click events on guides

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

add track ids to guide links
track links on click

### Benefits

Helps track click events on guide links

### Possible Drawbacks

None

### Applicable Issues

resolves MANAGER-1248

### CODE REVIEW

File: client/app/configuration/configuration.html [hunk 1]
```diff
@@ -39,14 +39,21 @@ <h2 class="oui-tile__title" data-ng-bind="'core_type_of_guide_' + key | translat
                                 <li class="oui-tile__item" data-ng-repeat="guide in value track by $index">
                                     <div class="oui-tile__definition">
                                         <a target="_blank"
+                                            data-track-on="click"
```
Comment (antleblanc) [inline client/app/configuration/configuration.html]: I think you can also add: `rel="noopener"` for both anchors.

Comment (varun257) [reply client/app/configuration/configuration.html]: have added

**Comment by antleblanc** [comment]
@varun257 In order to merge your pull request, could you please resolve existing conflict? Thanks

**Comment by jleveugle** [comment]
Hello @varun257,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: client/app/configuration/configuration.controller.js [hunk 2]
```diff
@@ -31,6 +35,18 @@ angular.module('App').controller(
       this.selectTypeOfGuide('domainHosting');
     }
 
+    buildSummitData() {
```
Comment (cbourgois) [inline client/app/configuration/configuration.controller.js]: I think you should rebase (code removed in https://github.com/ovh-ux/ovh-manager-web/commit/7e0c43aa5fc814a61f3cea8c325713b5cc8237ac)

Comment (varun257) [reply client/app/configuration/configuration.controller.js]: Thanks for the tip. Have rebased

**Pull request closed without merging**